app/core/security.py

```diff
@@ -0,0 +1,25 @@
+import secrets
+
+
+def password_validator(password: str) -> str:
+    """
+    Check the password strength, validity and remove trailing spaces.
+    This function is intended to be used as a Pydantic validator:
+    https://pydantic-docs.helpmanual.io/usage/validators/#reuse-validators
+    """
+    # TODO
```

Comment:
Remove this comment

Reply:
I think we should keep the TODO so we remember to add more checks like at least a number etc